src/utils/IO/system_checks.jl

```diff
@@ -38,17 +31,19 @@ end
 
 ### PV Bus Check ###
 
-function pvbuscheck(buses::Array{Bus}, generators::Array{T}) where {T<:Generator}
+function pv_bus_check(buses, generators)
     pv_list = -1*ones(Int64, length(generators))
-    for (ix,g) in enumerate(generators)
-        g.bus.bustype == "PV" ? pv_list[ix] = g.bus.number : continue
+    for (ix, gen) in enumerate(generators)
+        if gen.bus.bustype == PV::BusType
+            pv_list[ix] = gen.bus.number
+        end
     end
 
     for b in buses
-        if b.bustype == "PV"
-            b.number in pv_list ? continue : 0 #@warn "The bus ", b.number, " is declared as PV without a generator connected to it"
-        else
-            continue
+        if b.bustype == PV::BusType
```

Comment:
The old code effectively did nothing.  Do we want to do something here?

Reply:
This is not necessary anymore.

Reply:
Removed.